cliff.toml

```diff
@@ -0,0 +1,120 @@
+# configuration file for git-cliff (0.1.0)
+
+[changelog]
+# changelog header
+header = """
+# Changelog\n
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).\n
+"""
+# template for the changelog body
+# https://tera.netlify.app/docs/#introduction
+body = """
+{% if version %}\
+    ## Platform Version {{ version | trim_start_matches(pat="v") }} - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## Platform Version VERSION - UNRELEASED
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first | split(pat="\n") | first }}\
+    {% endfor %}
+{% endfor %}\n
+"""
+# remove the leading and trailing whitespace from the template
+trim = true
+# changelog footer
+footer = """
+<!-- generated by git-cliff -->
+"""
+
+[git]
+# parse the commits based on https://www.conventionalcommits.org
+conventional_commits = true 
+# filter out the commits that are not conventional
+filter_unconventional = false 
+
+commit_preprocessors = [
+    { pattern = "\\(#([0-9]+)\\)", replace = "([#${1}](https://github.com/infinyon/fluvio/issues/${1}))"}
+]
+
+# regex for parsing and grouping commits
+commit_parsers = [
+
+    { message = ".*[Bb]ump", group = "Noise", skip = true},
+    { message = ".*[Rr]evert", group = "Noise", skip = true},
+    { message = ".*[Cl]ippy", group = "Noise", skip = true},
+    { message = "^Merge pull request", group = "Noise", skip = true},
```

Comment:
Could you add a "[Ii]gnore.*" or "[Ss]kip.*", so we can skip a commit with that too?